src/aap_eda/core/migrations/0028_remove_activation_credentials_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.7 on 2024-03-25 19:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        (
+            "core",
+            "0027_credentialtype_alter_permission_resource_type_and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="activation",
+            name="credentials",
+        ),
+        migrations.RemoveField(
+            model_name="activation",
+            name="system_vault_credential",
+        ),
+        migrations.RemoveField(
```

Comment:
@hsong-rh If we did this the migration in create_initial_data will fail.

Reply:
@hsong-rh https://github.com/ansible/eda-server/blob/909c4b40b53d506a0f9004aa617ff25ddcc445e1/src/aap_eda/core/management/commands/create_initial_data.py#L435 this will fail